Ginger/GingerCoreNET/ActionsLib/ActSetVariableValue.cs

```diff
@@ -134,6 +136,49 @@ public override void Execute()
                 {
                     ((VariableDynamic)Var).ValueExpression = this.Value;
                 }
+                else if (Var.GetType() == typeof(VariableNumber))
+                {
+                    try
+                    {
+                        var varNumber = ((VariableNumber)Var);
+                        if (float.Parse(calculatedValue) >= float.Parse(varNumber.MinValue) && float.Parse(calculatedValue) <= float.Parse(varNumber.MaxValue))
+                        {
+                            varNumber.Value = calculatedValue;
+                        }
+                        else
+                        {
+                            Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
+                            Error = $"The value {calculatedValue} is not in the range, {Var.Name}:-[Min value:{varNumber.MinValue}, Max value:{varNumber.MinValue}]   {GingerDicser.GetTermResValue(eTermResKey.Variable) }.";
+                        }
+                    }
+                    catch (System.Exception ex)
+                    {
+                        Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
+                        Reporter.ToLog(eLogLevel.ERROR, "Error occured during SetValue for Variable number type..", ex);
+                    }
+
+                }
+                else if(Var.GetType() == typeof(VariableDateTime))
+                {
+                    var varDateTime = ((VariableDateTime)Var);
+                    try
+                    {
+                        if (DateTime.Parse(calculatedValue) >= DateTime.Parse(varDateTime.MinDateTime) && DateTime.Parse(calculatedValue) <= DateTime.Parse(varDateTime.MaxDateTime))
```

Comment:
Need to validate the format as well. If the format on the variable is DD/MM/YYYY and  in the action user provide incorrect format it should fail